modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/users/admin/management/toolbar/FilterContributorRegistryUtil.java

```diff
@@ -39,8 +37,18 @@ public FilterContributor[] getFilterContributors(String id) {
 		return filterContributors.toArray(new FilterContributor[0]);
 	}
 
-	@Activate
-	protected void activate(BundleContext bundleContext) {
+	private static final Log _log = LogFactoryUtil.getLog(
+		FilterContributorRegistryUtil.class);
+
+	private static final ServiceTrackerMap<String, List<FilterContributor>>
+		_serviceTrackerMap;
+
+	static {
+		Bundle bundle = FrameworkUtil.getBundle(
+			FilterContributorRegistryUtil.class);
+
+		BundleContext bundleContext = bundle.getBundleContext();
+
```

Comment:
Hey @GabrielHenriquedev, try to put a property here to replace the `filterContributor.getManagementToolbarKey()` usage and make the tracker use the property only.